README.md

```diff
@@ -1,16 +1,17 @@
 # studi.my: 101 Beginning Python
+
 The curriculum for studi.my: Beginning Python
 
-This is the curriculum material to learn basic Python programming.
+This is the syllabus for studi.my's basic Python programming.
```

Comment:
We don't need to specify the site studi.my as this was meant to be generic and usable by others as well.